testing_exercise/testing.js

```diff
@@ -0,0 +1,46 @@
+function replaceWith(str, rep, repWith) {
+	var newStr = "";
+	for (var i = 0; i < str.length; i++) {
+		if (str[i] === rep) {
+			newStr += repWith;
+		} else {
+			newStr += str[i];
+		}
+	}
+	return newStr;
+}
+
+function expand(arr, repeat) {
+	var expandedArr = [];
+	for (var i = 0; i < repeat; i++) {
+		expandedArr = expandedArr.concat(arr);
+	}
+	return expandedArr;
+}
+
+function acceptNumbersOnly() {
+	for (var i = 0; i < arguments.length; i++) {
+		if (typeof arguments[i] !== 'number' || isNaN(arguments[i])) {
+			return false;
+		}
+	}
+	return true;
+}
+
+function mergeArrays(arr1, arr2) {
+	if (Array.isArray(arr1) === false || Array.isArray(arr2) === false) {
+		return false;
+	}
+
+	return arr1.concat(arr2).sort();
```

Comment:
one gotcha to be aware of with `sort` - it won't sort numbers correctly! The default sort in JavaScript is alphabetical, so for example:

`[23,1,4].sort() // [1, 23, 4]`

The example doesn't catch this. 

To fix this, you can pass in a comparator to the sort function telling it how it should sort. [reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)